// const input = 10;

// try{
//     throw "Error from try block";
// }
// catch(error){
//     console.log(error);
//     console.log("from catch block");
// }
// finally{
//     console.log("I'm From Finally block");
// }

//Range Error Example 

// let number_1 = 1;
// try{
// let example = number_1.toPrecision(500);
// console.log(example);
// }
// catch(error){
//     console.log(error);
// }

//Reference Error Example

// let number_1 = 10;

// try{
//     number_1 = someNumber + 1;
// }
// catch(error){
//     console.log(error);
// }

//Syntax Error

// let number_1 = 10;

// try{
// console.log("number_1);
// }
// catch(error){
//     console.log(error);
// }

//Type Error

// let number_1 = 10;

// try{
//     number_1.toUpperCase();
// }
// catch(error){
//     console.log(error);
// }

